duo.sgnl.yaml

```diff
@@ -0,0 +1,243 @@
+description: "Duo as a System of Record"
+address: "api-{{Input Required: Duo Hostname}}.duosecurity.com" # https://duo.com/docs/adminapi#api-details
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "Duo-1.0.0"
+adapterConfig: "ewogImFwaVZlcnNpb24iOiAidjEiIAp9=" # {"apiVersion": "v1"}
+auth:
+  - basic: # https://duo.com/docs/adminapi#authentication
+      username: "{{Input Required: Duo Integration Key}}"
+      password: "{{Input Required: Duo Secret Key}}"
+entities:
+  User: # https://duo.com/docs/adminapi#retrieve-users
+    displayName: User
+    externalId: User
+    description: User entity in Duo
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 300
```

Comment:
Why is the pagesize 300 everywhere?

Reply:
Most endpoints accept a maximum page size of 300, although it's not standardized. There is no limit on the requested page size - eg. a page size of 99999 will be accepted but only return 300 values at most

Reply:
Let's stick to page size 100 as a standard. 
We can document the max allowed page size and possibly add validation on the adapter end if we know the limits on page size.